src/test/java/resources/account.feature

```diff
@@ -0,0 +1,6 @@
+Feature: Account Management test
+
+  Scenario: Account status API Test
+    Given I have tenant as "gorilla"
+    When I call accountStatus endpoint for "835322416" MSISDN, with status of 200
```

Comment:
how do you make sure that the MSISDN 835322416 exists in the DB?

Reply:
This is must have, you can check other And related test cases and reuse the code if it suites this usacase